experimental/packages/otlp-grpc-exporter-base/src/util.ts

```diff
@@ -120,14 +123,107 @@ export function validateAndNormalizeUrl(url: string): string {
       'URL path should not be set when using grpc, the path part of the URL will be ignored.'
     );
   }
-  if (target.protocol !== '' && !target.protocol?.match(/(http|grpc)s?/)) {
+  if (target.protocol !== '' && !target.protocol?.match(/(http)s?/)) {
```

Comment:
For an exact match, we should try 
```suggestion
  if (target.protocol !== '' && !target.protocol?.match(/^(http)s?:$/)) {
```

Or we may leak cases like `httpx:`